src/main/scala/com/danielasfregola/twitter4s/entities/GeoCoordinate.scala

```diff
@@ -0,0 +1,6 @@
+package com.danielasfregola.twitter4s.entities
+
+final case class GeoCoordinate(longitude: Double, latitude: Double) {
```

Comment:
We can also name this [GeoPosition](http://reference.wolfram.com/language/ref/GeoPosition.html) or [GeoLocation](https://developer.mozilla.org/en-US/docs/Web/API/Geolocation/Using_geolocation).
I decided to follow [Microsoft](https://msdn.microsoft.com/en-us/library/system.device.location.geocoordinate(v=vs.110).aspx) and name it GeoCoordinate. 

One thing to note though, it seems that the standard order for coordinate is (latitude, longitude). But Twitter seems to follow (longitude, latitude) order. This might confuse some people.

Reply:
I am very happy with the name you picked: I would stick with coordinate.

I wouldn't worry about people been confused: we are using a case class rather than a simple tuple exactly for that reason, so that can use named parameters.